wiki/Tournaments/UKCC/UKCC7/en.md

```diff
@@ -0,0 +1,636 @@
+- ***Where rules do not prevail, common sense shall. The administration has explicit discretion to apply their judgement on this as they see fit.***
+- Players are **REQUIRED** to join the public discord so that they are contactable by other players and staff. If a player can’t be contacted or is not responding to messages, through discord, ingame messages or forum pms, contact the staff.
+- Mappools are to be released the Monday before a stage is due to happen
+
+### Reschedules
+
+- Schedules for the next round are released on the Monday before a stage is due to happen and **ALL** matches will be scheduled for the weekend. You can reschedule for weekdays at the staffs discretion.
+- In order to reschedule, you must contact your opponents, agree on a time and be able to provide **PROOF** (by form of a screenshot/image) of the agreed reschedule to the Hosts.
+- In effort to ensure as many matches take place as possible, reschedules are allowed up to **1 HOUR BEFORE YOUR MATCH IS DUE TO BE PLAYED**. However, your opponent holds the right to **refuse a reschedule if asked less than 24 hours from the scheduled time**
+- If it is impossible to find a time to reschedule to that suits both players, both players should talk to the Hosts (Bae- or Yazzehh) and we will attempt to sort something out.
+- Matches that are uncertain to happen (for example, losers’ bracket matches) will still appear on the schedule. They are to be rescheduled if needed in the same way as confirmed matches.
+- If a match needs rescheduling to a time after a related losers’ bracket match, the matches relying on the outcome of the rescheduled match will be rescheduled by staff and 24 hours will be given for rescheduling them, if needed.
+
+## Links
+
+![Links](img/links.png)
+
+- [Discussion Thread](https://osu.ppy.sh/community/forums/topics/940042)
+- [UKCC Discord server](https://discord.gg/EtsU7Cw)
+- [Livestream](https://twitch.tv/osuukcc)
+- [Challonge bracket](https://challonge.com/ukcc7)
+- [Pick'em predictions website](https://pickem.hwchr.com/tournaments/15)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1ubY1Oci0HaH-rny6IKbqlRszPzquwSm9n4fwwnqn5yA/edit?usp=sharing)**
+
+## Rules
+
+![Rules](img/rules.png)
+
+- Referees will create the lobby approximately **10 minutes** before the match is due to start.
+- Players have a 10 minute grace period after their scheduled match time to join the lobby. If a player fails to join the lobby in this time, then the default win will go to the other player. If both players do not show up, both players will recieve losses.
+- We take no-shows seriously, if you fail to show up to your match without valid reason, you may forfeit your right to future UKCC tournaments.
+- The referee cannot play during the match.
+
+### __**NEW**: Qualifying Rounds__
```

Comment:
```suggestion
### Qualifying rounds
```
assuming "new" is because this wasn't in ukcc6, you could mention that in a sentence below this header